README.md

```diff
@@ -11,53 +11,52 @@ New Relic requires the following environment variables to identify Kubernetes ob
 - `NEW_RELIC_METADATA_KUBERNETES_POD_NAME`
 - `NEW_RELIC_METADATA_KUBERNETES_CONTAINER_NAME`
 
-These environment variables can be set manually by the customer, or they can be automatically injected using a MutatingAdmissionWebhook.
-New Relic provides an easy method for deploying this automatic approach.
+These environment variables can be set manually by the customer, or they can be automatically injected using a `MutatingAdmissionWebhook`.
```

Comment:
I feel the introduction located in [docs/performance.md](docs/performance.md) is easy and simpler: 

> The injection of New Relic APM Metadata is implemented as a webhook using Kubernetes MutatingAdmissionWebhook, which is in fact an AdmissionWebhook.
>
>Admission webhooks run as part of the Kubernetes API Request Lifecycle.
>
>An admission controller is a piece of code that intercepts requests to the Kubernetes API server prior to persistence of the object, but after the request is authenticated and authorized.
>
>--- kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-are-they